### ISSUE TITLE
java.lang.ArithmeticException: / by zero - when scaling VR 

### ISSUE THREAD

**Issue by borisstoyanov** [apache/cloudstack#4343]
##### ISSUE TYPE

 * Bug Report

##### COMPONENT NAME

~~~
VR Scaling
~~~

##### CLOUDSTACK VERSION

~~~
4.11.3
~~~

##### CONFIGURATION

with VMWARE 6.5

##### STEPS TO REPRODUCE

~~~
Version used: 4.11.3

Steps: 
1. Set the new system offering with 2 cores instead of 1 and fill in the same details as the default one. 
2. Scale the VR to the new Offering

2020-09-23 13:26:51,689 ERROR [c.c.h.v.r.VmwareResource] (DirectAgent-132:ctx-85f479ac 10.10.2.140, job-306/job-310, cmd: ScaleVmCommand) (logid:3407676c) Unexpected exception:
java.lang.ArithmeticException: / by zero
~~~

**Comment by rhtyd** [comment]
ping @shwstppr - do you remember if that's fixed?

**Comment by shwstppr** [comment]
no idea @rhtyd. I'll check

**Comment by shwstppr** [comment]
@rhtyd I've checked and was able to reproduce the issue on latest master as well. I've created a draft PR for fixes. Adding fix 4.14 is okay or do we need it for any previous release as well? cc @borisstoyanov

**Comment by rhtyd** [comment]
4.14+ is okay
